tests/test_query_parsing.py

```diff
@@ -11,25 +11,43 @@
 
 DB_URL = "sqlite+aiosqlite://"
 
+# normal query extraction parameters
+QUERY_STR = 'query_str'
```

Comment:
This entire thing should probably be an Enumerate

Reply:
Okay, sure.  Changing to an Enum

Reply:
@ddkang Done with the suggested changes